fineract-progressive-loan/src/main/java/org/apache/fineract/portfolio/loanproduct/calc/ProgressiveEMICalculator.java

```diff
@@ -48,18 +49,25 @@
 @DependsOn("moneyHelper")
 public final class ProgressiveEMICalculator implements EMICalculator {
 
+    private MathContext mc;
+
     private static final BigDecimal DIVISOR_100 = new BigDecimal("100");
     private static final BigDecimal ONE_WEEK_IN_DAYS = BigDecimal.valueOf(7);
 
+    @Override
+    public void setMathContext(final MathContext mc) {
```

Comment:
We dont need this. EmiCalculator always work with `ProgressiveLoanInterestScheduleModel` and it already has the MathContext

Reply:
Fixed.